src/controllers/authControllers.ts

```diff
@@ -40,13 +57,15 @@ class AuthController {
       const token = user.generateAuthToken();
 
       return res.send(token);
```

Comment:
try to use res.status(code).json({ message: 'this is a message', data: {}})
code: 
200 for getting, updating resource
201: for creating resource
422: Validation error
400: Bad request
401: Unauthorised operation
403:------
404: Not found error
@iamChukka

Reply:
you can create like a constant file where you put the code inside an object an export to be used everywhere. Reusability pattern